x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -531,14 +542,15 @@ private void indexBwcInvalidation(UserToken userToken, ActionListener<Boolean> l
                     .request();
             final String tokenDocId = getTokenDocumentId(userToken);
             final Version version = userToken.getVersion();
-            securityIndex.prepareIndexIfNeededThenExecute(listener::onFailure, () ->
+            securityIndex.prepareIndexIfNeededThenExecute(ex -> listener.onFailure(traceLog("prepare index", tokenDocId, ex)), () ->
```

Comment:
Maybe keep one of `prepare index` or `prepare security` for consistency ? Or merge them to `prepare security index` ? I feel these identifiers offer little assistance to an ES Admin troubleshooting failures, but will be extremely helpful when we need to look into an issue. 
To be honest, I don't have any good ideas on how these could become more helpful for ES admins without being extremely descriptive